src/writer/async_tokio.rs

```diff
@@ -340,3 +484,99 @@ mod indentation_async {
         );
     }
 }
+
+#[cfg(test)]
+mod element_writer {
```

Comment:
I supposed that this module will be inside the `tests` module, but that's also OK